sdk/storage/inc/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -164,9 +166,9 @@ namespace Azure { namespace Storage { namespace DataLake {
 
   struct SetAccessControlRecursiveResponse
   {
-    int32_t DirectoriesSuccessful = int32_t();
-    int32_t FilesSuccessful = int32_t();
-    int32_t FailureCount = int32_t();
+    int32_t DirectoriesSuccessful;
+    int32_t FilesSuccessful;
+    int32_t FailureCount;
```

Comment:
won't compiler give warnings for these uninitialized variables?

Reply:
I don't see any warnings from my side. Also, CI passes.

Reply:
fine, if compilers say it's okay, then i'm okay.